src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details_decision_hep.html

```diff
@@ -1,72 +1,77 @@
-<p class="automatic-decision">Automatic Decision: <span
-  ng-class="{'accept_core': vm.record._extra_data.relevance_prediction.decision == 'CORE',
-              'accept': vm.record._extra_data.relevance_prediction.decision == 'Non-CORE',
-              'reject': vm.record._extra_data.relevance_prediction.decision == 'Rejected'}"
-  ng-bind="vm.record._extra_data.relevance_prediction.decision + ' ' + (vm.record._extra_data.relevance_prediction.max_score | number : 2)"></span>
-</p>
+<div ng-if="!vm.record.metadata.inspire_categories.length && !vm.record.metadata.arxiv_categories.length">
+  At least one subject is required
+</div>
+<div ng-if="vm.record.metadata.inspire_categories.length > 0 || vm.record.metadata.arxiv_categories.length > 0">
```

Comment:
I'm checking for the length of both, `arxiv_categories` and `inspire_categories` because the entries in both are enlisted in the `Subject Areas` [template](https://github.com/inspirehep/inspirehep-search-js/blob/master/src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/details/details_subject_areas.html#L19).

Reply:
it's not really needed, as the workflow has some step to derive inspire categories from arXiv ones. So if there are arXiv categories, there should always be inspire categories.